src/traits/networking/libp2p_network.rs

```diff
@@ -114,7 +114,7 @@ impl<
         TYPES: NodeType,
         PROPOSAL: ProposalType<NodeType = TYPES>,
         VOTE: VoteType<TYPES>,
-        ELECTION: Election<TYPES>,
+        ELECTION: Membership<TYPES>,
```

Comment:
Nit: Can we change `ELECTION` to `MEMBERSHIP` for `TestableNetworkingImplementation`, `Libp2pCommChannel`, and perhaps some other similar traits/structs?

It's probably easier to just replace all `ELECTION` and `election` in the codebase.